contracts/base/LoansBase.sol

```diff
@@ -494,10 +494,57 @@ contract LoansBase is LoansInterface, Base {
         address payable recipient
```

Comment:
We should change this function be called `_payOutLiquidator` and to remove the `amount` parameter since we will need to calculate it here.

Reply:
Done